vllm/model_executor/models/llava.py

```diff
@@ -149,14 +149,15 @@ def __init__(self,
                                                 config.vocab_size, logit_scale)
         self.sampler = Sampler()
 
-    def _validate_image_data(self, data: torch.Tensor) -> torch.Tensor:
-        if list(data.shape)[1:] != [
-                3, self.config.vision_config.image_size,
-                self.config.vision_config.image_size
-        ]:
+    def _validate_pixel_values(self, data: torch.Tensor) -> torch.Tensor:
+        h = w = self.config.vision_config.image_size
+        expected_dims = (3, h, w)
+        actual_dims = tuple(data.shape[1:])
+
+        if actual_dims != expected_dims:
             raise ValueError(
-                "The expected image tensor shape is batch dimension plus "
-                "channel, height and width.")
+                "The expected shape of pixel values is batch dimension plus "
+                f"{actual_dims}. You supplied {tuple(data.shape)}.")
```

Comment:
This should be `expected_dims`, no?

Also - can you make this more explicit? Something like
```
f"The expected shape of pixel values is (batch size, 3, {h}, {w}). You supplied {tuple(data.shape)}.".
```

Reply:
Will update